src/standardeditingmodeediting.js

```diff
@@ -54,5 +54,12 @@ export default class StandardEditingModeEditing extends Plugin {
 		} );
 
 		editor.commands.add( 'restrictedEditingException', new RestrictedEditingExceptionCommand( editor ) );
+
+		// Set unique class to every $root element for additional styling in standard editing
```

Comment:
```suggestion
		// Make styling restricted and standard modes differently possible.
```

Don't read the code in a comment. Just say why it's there (if that's even necessary).

Reply:
I removed this comment entirely.